cupy/math/window.py

```diff
@@ -28,13 +31,21 @@ def blackman(M):
 
     .. seealso:: :func:`numpy.blackman`
     """
-    if M < 1:
-        return from_data.array([])
     if M == 1:
-        return basic.ones(1, float)
-    n = ranges.arange(0, M)
-    return 0.42 - 0.5 * trigonometric.cos(2.0 * numpy.pi * n / (M - 1))\
-        + 0.08 * trigonometric.cos(4.0 * numpy.pi * n / (M - 1))
+        return cupy.creation.basic.ones(1, float)
+    if M <= 0:
+        return cupy.creation.from_data.array([])
```

Comment:
Please use `cupy` module.
```suggestion
        return cupy.array([])
```